sources/Common/include/Options.h

```diff
@@ -52,7 +52,8 @@ class Options {
   enum class Request {
     RUN_SIMULATION = 0,  ///< Run simulation
     HELP,                ///< help display is requested
-    VERSION              ///< version display is requested
+    VERSION,             ///< version display is requested
+    STEADY_STATE         ///< steady state calibration is requested
```

Comment:
I'd avoid the term "calibration", I'd simply say calculation or stabilization

Reply:
I reverted to "calculation".